cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -45,11 +45,14 @@ def setUp(self):
         self.user = UserFactory()
         self.factory = RequestFactory()
         self.request = self.factory.get('/course')
+
+        # pylint: disable=protected-access
+        self.split_store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.split)
```

Comment:
I'm not sure why you need `self.split_store` here. It seems to me that the default modulestore to create a course with should be the split modulestore. I think that's determined by the ordering of their configurations.

My question is why is this change being made in this file rather than in the definition of `ModuleStoreTestCase`: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/django_utils.py#L408